library/cmake/Modules/FindPHP.cmake

```diff
@@ -87,26 +101,29 @@ MARK_AS_ADVANCED(
 	PHP_VERSION_STRING
 )
 
+set(PHP_DEVELOPMENT_COMPONENT_ACCPETED TRUE)
+if(PHP_USE_DEVELOPMENT_COMPONENT)
+	FIND_PATH(zend_path
+		NAMES "zend.h"
+		PATHS ${PHP_INCLUDE_DIRS}
+	)
+	if(zend_path)
+		set(PHP_DEVELOPMENT_COMPONENT_FOUND TRUE)
+	else()
+		set(PHP_DEVELOPMENT_COMPONENT_ACCPETED FALSE)
+	endif()
+endif()
+
 INCLUDE(FindPackageHandleStandardArgs)
 FIND_PACKAGE_HANDLE_STANDARD_ARGS(
 	php
 	DEFAULT_MSG
-	PHP_VERSION_STRING
```

Comment:
I think the reason for the blank line is, that you accidentally removed PHP_VERSION_STRING

Reply:
Actually, the blank line was mostly intended for separating the variables that all needed to be true to set `PHP_FOUND` to true -- the removal of `PHP_VERSION_STRING` was intentional since it was in the wrong part of the order (and included twice).  😛

Reply:
> [...] it was in the wrong part of the order (and included twice)

I see.